test/cli/test_login.rb

```diff
@@ -0,0 +1,237 @@
+require_relative "test_helper"
+require "aruba"
+
+class CLITest < Minitest::Test
```

Comment:
Give this class a unique name, or we'll probably end up clobbering multiple methods across the test suite. Maybe something like `CLILoginTest`?